pkg/collector/corechecks/containers/cri/adapter.go

```diff
@@ -13,11 +13,11 @@ import (
 )
 
 var metricsNameMapping = map[string]string{
-	"container.uptime":     "cri.uptime",
-	"container.cpu.usage":  "cri.cpu.usage",
-	"container.memory.rss": "cri.mem.rss",
```

Comment:
what about linux? shouldn't we have a `container.memory.rss` metric on linux?

Reply:
Not from all collectors (`cri` collector for instance). The closest behaviour seems to be to use `UsageTotal` which should always be reported.